src/util/expr.cpp

```diff
@@ -19,13 +20,20 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <stack>
 
+/// Move the given argument to the end of `exprt`'s operands.
+/// The argument is destroyed and mutated to a reference to a nil `irept`.
+/// \param expr: `exprt` to append to the operands
```

Comment:
Should that be `\ref exprt` instead? (But maybe doxygen treats "`exprt`" the same.) To be considered throughout this PR.

Reply:
There are treated differently. The backward quotes only enforces monospace. Would you prefer seeing a link everywhere? I was worried it would make the source less readable while not adding much to the doxygen (where there are already links to `exprt` everywhere). What do you think?

Reply:
I would have used the human readable "expression" instead of the programmer-only "`exprt`" (and, equally, would have said "to the end of the expression's operands" in the first sentence). But then that's all a debate about style and therefore entirely up to you - unless there is broad consensus and consistency across the entire documentation.

Reply:
There's no broad consensus, but I think I saw the the reverse-quoted version more frequently. So I'll leave it that way.